storybookjs/storybook
# Core: Allow custom postcss config

- Search for the nearest postcss config
- Maintain previous defaults if no postcss config

Issue: #2455
Old PR: #4411

## What I did
- Add `find-up` to dependencies to search for the nearest postcss.config.js
- Use the nearest postcss config (`.postcssrc.json`, `postcss.config.js`...) to determine postcss-loader options if present, maintain previous defaults if not.

## How to test
Run the example `storybook/examples/postcss-kitchen-sink`. 
The `h1` should have the `color: #639;` instead of `color: rebeccapurple;`.

- Is this testable with Jest or Chromatic screenshots? 
No
- Does this need a new example in the kitchen sink apps? 
I created one, not sure if it was mandatory.
- Does this need an update to the documentation?
Yes, I added a documentation in `/custom-postcss-config`
